styles/mixins/_layout.scss

```diff
@@ -0,0 +1,20 @@
+@use '../variables' as var;
+
+$-layout-space: var.$layout-space;
+
+/**
+ * Establish a vertical (margin) rhythm for this element's immediate
+ * children (i.e. put equal space between children).
+ *
+ * This mixin uses `!important` such that it can compete with specificity
+ * of reset rules that set some of our element's margins to 0. That allows
+ * these rules—which are applied to a parent element—to be able to assert
+ * margins, as it should be able to do.
+ *
+ * @param $size [$layout-space]: Spacing size (padding)
+ */
+@mixin vertical-rhythm($size: $-layout-space) {
+  & > * + * {
```

Comment:
Maybe a comment would help to know the meaning of this CSS selector? I think this means to apply the rule to all the children that have siblings elements.

Reply:
Sure, that's a so-called _lobotomized owl_ selector; I can add a link to a resource that explains it.